core/src/main/resources/lib/hudson/rssBar_tr.properties

```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Legend=Gösterge
-for\ all=tümü için
-for\ failures=tüm başarısız durumlar için
-for\ just\ latest\ builds=sadece son yapılandırma için
+legend=İkonlar hakkında
+forAll=(tümü)
+forFailures=(başarısız olanlar)
```

Comment:
are these brackets needed? the english version doesn't have them

Reply:
It's currently not possible to have grammatically correct translation for these strings. Using parenthesis I was able to build correct representation. 

I'm pretty satisfied with the result however if wanted I can work on a follow up PR refactoring original strings to allow better localization.